lib/elements/elements.css

```diff
@@ -216,8 +216,66 @@
   top: 72px;
 }
 
+.modal {
```

Comment:
These styles are defining a glass pane behind the model dialog, rather than a modal dialog itself? Perhaps calling it something like `.modalpane` or `.modallayer`?